towhee/engine/__init__.py

```diff
@@ -22,4 +22,6 @@
 MOCK_PIPELINES = str(Path(__file__).parent.parent /
                      'tests/test_util/resnet50_embedding.yaml')
 
+# These are used for loading defaults into .towhee
 MOCK_OPS = str(Path(__file__).parent.parent / 'tests/mock_operators')
+MOCK_PIPES = str(Path(__file__).parent.parent / 'tests/mock_pipelines')
```

Comment:
We might remove `mock_piplines` and `mock_operators` once hub is all set, so we might need to change this in the future.